installers/Windows/req-extras-pull-request.txt

```diff
@@ -17,9 +17,9 @@ matplotlib
 cython
 sympy
 
+# Spyder external plugins
+spyder-terminal>=1.2.1
+
 # Spyder external dependencies (spyder-kernels and qdarkstyle)
 ./external-deps/spyder-kernels
 ./external-deps/qdarkstyle
-
-# There are no wheels for version 3.3
-cryptography==3.2.1
```

Comment:
Did you check that the installers from this PR work without this constraint?

Reply:
I checked the full version installation. It opens normaly and I can see the spyder terminal plugin. Is there something else I should test?